### PR TITLE
 Display errors properly shown in SAML application creation window

### PR DESCRIPTION
**Pull request by kayathiri4**
### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
- None

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -1712,7 +1713,8 @@ export const console: ConsoleNS = {
                                 validations: {
                                     empty: "Please provide the meta file url",
                                     invalid: "Enter a valid URL"
-                                }
+                                },
+                                errorMessage: "The metadata Url is invalid"
```
Comment (VivekVinushanth) [inline modules/i18n/src/translations/en-US/portals/console.ts:1717]: can we stick to `URL` ?

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 2]
```diff
@@ -315,6 +322,24 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
                     return;
                 }
 
+                if (error.response && error.response.data && error.response.data.code &&
+                    error.response.data.code === "SAML-60002") {
```
Comment (inthirakumaaran) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:326]: Shall we get them into constant?

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 3]
```diff
@@ -396,6 +424,70 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
...
+                metaUrlElement.blur();
+                break;
+            }
+        }
+    };
+
+    /**
+     * Handles the protocol form values change.
+     *
+     * @param state
```
Comment (inthirakumaaran) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:464]: Add the param description, check the other places as well

File: apps/console/src/features/applications/components/wizard/saml-protocol-settings-all-option-wizard-form.tsx [hunk 4]
```diff
@@ -443,6 +467,8 @@ export const SAMLProtocolAllSettingsWizardForm: FunctionComponent<SAMLProtocolAl
                     <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }>
                         <Field
                             name="url"
+                            ref={ metaUrlRef }
+                            displayErrorOn={ "blur" }
```
Comment (thivi) [inline apps/console/src/features/applications/components/wizard/saml-protocol-settings-all-option-wizard-form.tsx:471]:
```suggestion
                            displayErrorOn="blur"
```

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 5]
```diff
@@ -315,6 +322,24 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
                     return;
                 }
 
+                if (error.response && error.response.data && error.response.data.code &&
+                    error.response.data.code === ApplicationManagementConstants.ERROR_CODE_ISSUER_EXISTS) {
+                    if (protocolValuesChange) {
+                        handleError("issuer", true);
+                        scrollToInValidField("issuer");
+                    }
+                    return;
```
Comment (thivi) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:331]:
```suggestion

                    return;
```

File: apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx [hunk 6]
```diff
@@ -315,6 +322,24 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
...
+                }
+
+                if (error.response && error.response.data && error.response.data.code &&
+                    error.response.data.code === ApplicationManagementConstants.ERROR_CODE_INVALID_METADATA_URL) {
+                    if (protocolValuesChange) {
+                        handleError("metaUrl", true);
+                        scrollToInValidField("metaUrl");
+                    }
+                    return;
```
Comment (thivi) [inline apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx:340]:
```suggestion

                    return;
```

**Review by thivi — APPROVED** [review]

**Review by sachiniWettasinghe — APPROVED** [review]

**Pull request merged**